queries/orderbook/order_quotes.sql

```diff
@@ -0,0 +1,9 @@
+with winning_quotes as (SELECT concat('0x', encode(oq.solver, 'hex')) as solver,
```

Comment:
Not claiming anything is wrong with that, but somehow I would feel safer to join also with the `orders` table on `uid`, and ask explicitly that the `class` of the order is `market`.

Will need to think if limit orders create any issues with the accounting here. (it really depends on what the `order_quotes` table contains for such orders).

Reply:
I have addressed this comment (visible a couple of lines below).